modules/button/react/lib/ToolbarDropdownButton.tsx

```diff
@@ -0,0 +1,89 @@
+/** @jsx jsx */
+import {jsx} from '@emotion/core';
+import {colors, spacing, borderRadius} from '@workday/canvas-kit-react-core';
+import {focusRing, useTheme, Themeable, EmotionCanvasTheme} from '@workday/canvas-kit-react-common';
+import {SystemIcon} from '@workday/canvas-kit-react-icon';
+import {ButtonColors} from './types';
+import {ButtonContainer} from './parts';
+import {IconButtonProps} from './IconButton';
+import {chevronDownSmallIcon} from '@workday/canvas-system-icons-web';
+
+export interface ToolbarDropdownButtonProps
+  extends Omit<IconButtonProps, 'size' | 'variant' | 'toggled' | 'onToggleChange'>,
+    Themeable {}
+
+const ToolbarDropdownButton = ({
+  theme = useTheme(),
+  buttonRef,
+  'aria-label': iconArialabel,
+  icon,
+  children,
+  ...elemProps
+}: ToolbarDropdownButtonProps) => {
+  const containerStyles = {
+    padding: 0,
+    minWidth: spacing.l,
+    width: 'auto',
+    height: spacing.l,
+    borderRadius: borderRadius.m,
+    'span > *': {
+      margin: 0,
+    },
+    ['& .wd-icon']: {
+      display: 'inline-block',
+      verticalAlign: 'middle',
+      width: '20px',
+      height: '20px',
+    },
+  };
+
+  const chevronStyles = {
+    margin: '0 2px 0 0',
+  };
+
+  const customIconStyles = {
+    marginLeft: '2px',
+    marginRight: 0,
+  };
```

Comment:
I needed these styles for the right styling per design, I can sync up with design again to see if this is necessary